### PR TITLE
Added EditorConfig for easier convention enforcement.

### PR DESCRIPTION
**Pull request by carldebilly**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
+root = true
```
Comment (dr1rrb) [inline .editorconfig]: Tu ne devrais pas le mettre

Comment (carldebilly) [reply .editorconfig]: Exactly.  That's exactly the best-practice for this feature.

Comment (dr1rrb) [reply .editorconfig]: Indeed ... if you are effectively at the root. For instance this prevent user to define configuration for all their repositories at once, which may configure some file types that you dit not configure.

Comment (carldebilly) [reply .editorconfig]: That's the root of the repository. It's supposed to be completed and the content shouldn't be impacted by something external.

**Pull request merged**